xml/ay_bootloader.xml

```diff
@@ -264,13 +264,27 @@
     <term>password</term>
     <listitem>
      <para>
-      If defined, then it protects boot via password. It has three subelements.
-      <literal>value</literal> holds password value. It can be either plain text
-      or already encrypted one. If <literal>encrypted</literal> is set to <literal>
-      true</literal> it means that <literal>value</literal> is already encrypted
-      password. <literal>unrestricted</literal> is to set unrestricted option for
-      password protection. <phrase os="sles;sled">For more information on boot passwords,
-      see <xref linkend="vle-grub2-yast2-boot-password"/>.</phrase>
+      If this is defined, it protects the boot loader with a password. The
+      system will not boot until the password is entered.
+    </para>
+    <para>
+      It has three subelements: <literal>value</literal>, 
+      <literal>encrypted</literal>, and <literal>unrestricted</literal>.
+    </para>
+    <para>
+      <literal>value</literal> holds the password. It can be either 
+      plain text, which &yast; will encrypt, or an already encrypted 
+      password. Set <literal>encrypted</literal> to <literal>true</literal> when you use an already encrypted password. 
+    </para>
+    <para>When <literal>unrestricted</literal> is set to 
```

Comment:
Hi, sorry I was absent for a while, I could not review your changes. This is not completely correct. It would be:

"when unrestricted is set to "false", users need the password defined as <value> in order to boot or edit grub menu entries. When it is set to "true", users can boot the system without a password, but need a password to edit grub menu entries.
If the option is omitted, it defaults to "true".

For the encryption thing, I think we could make it more precise, as customers will likely have no idea what we mean when we say they can use an "already encrypted password". It took me a while to figure it out myself. I would suggest this:

 "value" holds the password. It can be either 
 plain text, which &yast; will encrypt, _or a password already encrypted using grub-mkpasswd-pbkdf2_.
 Set <literal>encrypted</literal> to <literal>true</literal> when you use an already encrypted password. 
If the option is omitted, it defaults to "false".

Reply:
excellent, thank you!